doozerlib/plashet.py

```diff
@@ -170,11 +170,39 @@ def from_image_member_deps(self, el_version: int, assembly: str, releases_config
             dep_builds = self._get_builds(dep_nvr_list)
             missing_nvrs = [nvr for nvr, build in zip(dep_nvr_list, dep_builds) if not build]
             if missing_nvrs:
-                raise IOError(f"The following group dependency NVRs don't exist: {missing_nvrs}")
+                raise IOError(f"The following image member dependency NVRs don't exist: {missing_nvrs}")
             # Make sure image member dependencies have no ART managed rpms.
             art_rpms_in_deps = {dep_build["name"] for dep_build in dep_builds} & {meta.rpm_name for meta in rpm_map.values()}
             if art_rpms_in_deps:
                 raise ValueError(f"Unable to build plashet. Image member dependencies cannot have ART managed RPMs: {art_rpms_in_deps}")
             for dep_build in dep_builds:
                 component_builds[dep_build["name"]] = dep_build
         return component_builds
+
+    def from_rhcos_deps(self, el_version: int, assembly: str, releases_config: Model, rpm_map: Dict[str, RPMMetadata]):
```

Comment:
```suggestion
    def from_rhcos_deps(self, el_version: int, assembly: str, releases_config: Model, rpm_map: Dict[str, Dict]):
```
Just brew build dicts, right?